### DISCUSSION TITLE
eqMac v1.0.0

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#508] [General]
# v1.0.0 - Biggest Update yet
* Expert EQ with Unlimited Bands + Bandwidth + Precise Controls
* Spectrum Analyzer (as part of the Expert EQ)
* [AutoEQ](https://github.com/jaakkopasanen/AutoEq) Integration (part of Expert EQ)
* Enable/Disable Volume Boost
* Support for all Sampling Rates
* Fixed Offline access
* Fixed HDMI and USB Volume/Balance Controls
* Fixed slow memory leak
* Fixed MultiOutput and Aggregate Device Support
* Fixed many other bugs and crashes (not all of them but they will be fixed soon)

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.0.0'>eqMac v1.0.0</a>.</em>

**Comment by nodeful** [comment]
Sorry found a mistake in the install script, just fixing up and will publish the binary

**Reply by BenjaminHCCarr** [reply]
thanks, was wondering where the `.pkg` went

congrats on hitting 1.0!

**Reply by nodeful** [reply]
I found another issue with the release, so fixinf that now. Sorry for the confusion... Not my day today 😅

**Reply by nodeful** [reply]
I have now uploaded the right binary. Please try again

**Reply by DaniBedz** [reply]
Fantastic work! It seems a lot more stable than the last release. Thank you so much for your efforts. 👍

**Comment by henriw1410** [comment]
Update 1.0.0 installed flowless :  Balance control now ROCKS PERFECTLY. It was my only issue. Thank you for your fantastic work!

**Comment by JARVIS-AI** [comment]
Should we install **eqMac_UPDATE.zip** after eqMac.pkg ?

**Reply by nodeful** [reply]
No, this is just a file for the auto update. Sorry for the confusion i will try to make it clearer
